controllers/suite_test.go

```diff
@@ -77,3 +119,224 @@ var _ = AfterSuite(func() {
 	err := testEnv.Stop()
 	Expect(err).ToNot(HaveOccurred())
 })
+
+type MockSpireService struct {
+	entriesById        map[string]*common.RegistrationEntry
+}
+
+func NewMockSpireService() MockSpireService {
+	return MockSpireService{
+		entriesById:        map[string]*common.RegistrationEntry{},
+	}
+}
+
+func (m MockSpireService) getMatchingEntry(in *common.RegistrationEntry) (*common.RegistrationEntry, error) {
+nextEntry:
+	for _, entry := range m.entriesById {
+		if entry.SpiffeId != in.SpiffeId {
+			continue nextEntry
```

Comment:
I don't get this - wouldn't this loop endlessly? Assuming `nextEntry` is a label. Can `continue` without a label just work?

Reply:
continue labels are not like goto labels:
https://golang.org/ref/spec#Continue_statements

The last condition in this for is inside a nested for so needed the label. Since the label exists, I felt it was cleaner to have all the continues use it (rather than just the last one that needs it.)

Reply:
Thanks! I was not familiar with the go way of continue.